packages/core/src/components/popover/popover.tsx

```diff
@@ -116,8 +116,7 @@ export class Popover extends AbstractPureComponent2<IPopoverProps, IPopoverState
         minimal: false,
         modifiers: {},
         openOnTargetFocus: true,
-        // N.B. we don't set a default for `placement` here because that would override
-        // the deprecated `position` prop
+        // N.B. we don't set a default for `placement` here because that would override the `position` prop
         position: "auto",
```

Comment:
this will cause errors when `placement` is specified, which should be valid. we should configure this default inline during the render phase, not here as a static default prop